src/Suave/Utils/Bytes.fs

```diff
@@ -1,14 +1,17 @@
 ﻿namespace Suave.Utils
 
-module Bytes =
-  open System
-  open System.IO
-  open System.Text
-  open Suave.Async
+open System
+open System.IO
+open System.Text
+open Suave.Utils.Async
+
+
+
+module (* internal *) Bytes =
```

Comment:
I ran into trouble with BufferSegment being exposed in some public API (which can probably also be made internal).  Let's use a separate PR for this whole "internal" issue.